Xamarin.Forms.Core/ScrollView.cs

```diff
@@ -102,7 +102,7 @@ public Point GetScrollPositionForElement(VisualElement item, ScrollToPosition po
 
 			if (position == ScrollToPosition.MakeVisible)
 			{
-				bool isItemVisible = ScrollX < y && ScrollY + Height > y;
+				bool isItemVisible = ScrollX < x && ScrollX + Width > x + item.Width && ScrollY < y && ScrollY + Height > y + item.Height;
```

Comment:
This very difficult to read.
I think that it should be divided into several parts.
```C#
bool isVisibleHorizontal = x < ScrollX && ScrollX > x + item.Width - Width;
bool isVisibleVertical = y < ScrollY && ScrollY > y + item.Height - Height;
bool isItemVisible = isVisibleHorizontal && isVisibleVertical;
```

Reply:
agree with @paymicro let's make that change